.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can delete the `.vs` folder and add it to the `.gitignore` at the root of the repository.